lib/matplotlib/tests/test_colorbar.py

```diff
@@ -827,3 +827,30 @@ def test_aspects():
     np.testing.assert_almost_equal(
         cb[1][0].ax.get_position(original=False).height * 2,
         cb[1][2].ax.get_position(original=False).height, decimal=2)
+
+
+@image_comparison(['proportional_colorbars.png'], remove_text=True,
+                  style='mpl20')
+def test_proportional_colorbars():
+
+    x = y = np.arange(-3.0, 3.01, 0.025)
```

Comment:
```suggestion
    x = y = np.linspace(-3, 3, 241)
```

`np.arange` with non-integer step is generally not recommended

https://numpy.org/doc/stable/reference/generated/numpy.arange.html: 
> When using a non-integer step, such as 0.1, the results will often not be consistent. It is better to use numpy.linspace for these cases.

While it doesn't matter for the test, let's set a good example.

Reply:
I guess I don't understand this advice.  _Very_ often one wants to specify dx instead of N.  Sure you can compute one from the other, but I don't see why one would be more consistent than the other.

Reply:
Since you have limited numeric precision, the results are only approximately the same. For `arange` you often don't get expected "exact" numbers. e.g. the middle value is 0 with `linspace` but 1.06e-14 with `arange`. Additionally, the "endpoint not included" feature makes sense for indices, but usually not for number ranges. And in conjunction with the numeric jitter, this can have surprising effects; e.g. https://stackoverflow.com/questions/10011302/python-numpy-arange-unexpected-results.

Reply:
Sure, that's just floating point arithmetic.  I'm not convinced that makes arange bad style or "inconsistent"

Reply:
Well, "inconsistent" *is* numpy's description. That may not be the best term.

However, that the endpoint and number of points depend on small numeric effects is highly problematic. Practically, you don't know what you will get if endpoint is start + N times interval.

Anyway, I'm not going to argue more about this within a test.